x-pack/plugins/apm/server/types.ts

```diff
@@ -83,6 +84,7 @@ export interface APMPluginSetupDependencies {
   spaces?: SpacesPluginSetup;
   taskManager?: TaskManagerSetupContract;
   usageCollection?: UsageCollectionSetup;
+  infra?: InfraPluginSetup;
```

Comment:
Infra is a required plugin in `kibana.json`

Reply:
good point. 

according to [requiredPlugins](https://github.com/elastic/kibana/blob/8.2/docs/development/core/server/kibana-plugin-core-server.pluginmanifest.requiredplugins.md)

```
An optional list of the other plugins that **must be** installed and enabled for this plugin to function properly.
```
we can make it required